Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -169,7 +150,11 @@ def fetch_indicators(client: Client, indicator_type_lower: str, limit: int = -1)
                     "rawJSON": raw_data,
                     "fields": {
                         "port": argToList(item.get('tcpPorts', '')),
-                        "subfeed": item.get('serviceArea', '')
+                        "subfeed": item.get('serviceArea', ''),
```

Comment:
Aren't you changing `subfeed` to `service` in your other PR? Worthwhile to preemptively change it here even if that PR hasn't been merged yet.